test/repositories/formatters/PublishedProcessFormatterSpec.scala

```diff
@@ -16,23 +16,28 @@
 
 package repositories.formatters
 
-import play.api.libs.json.{JsError, JsObject, JsSuccess, Json}
-
-import models.PublishedProcess
+import java.time.LocalDateTime
 
 import base.UnitSpec
+import models.PublishedProcess
+import play.api.libs.json.{JsError, JsObject, JsSuccess, Json}
+import repositories.formatters.PublishedProcessFormatter.mongoFormat
 
 class PublishedProcessFormatterSpec extends UnitSpec {
 
   private val process: JsObject = Json.obj()
   private val id: String = "ext90002"
 
-  private val publishedProcess: PublishedProcess = PublishedProcess(id, process)
+  private val datePublished = LocalDateTime.of(2020, 1, 1, 12, 0, 1)
+
+  private val publishedProcess: PublishedProcess = PublishedProcess(id, 1, datePublished, process)
 
   private val json = Json.parse(
     s"""
        |{
        | "_id": "$id",
+       | "version": 1,
+       | "datePublished": "2020-01-01T12:00:01",
```

Comment:
If it is not too difficult it might be better to create date here from val defined above. This should elminate any potential GMT/BST issues.

Reply:
Will be dealt with when moving from joda.time to Java time classes.